frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -318,7 +318,13 @@ end
 function ReaderHighlight:genHighlightDrawerMenu()
     local get_highlight_style = function(style)
         return {
-            text = highlight_style[style],
+            text_func = function()
+                local text = highlight_style[style]
+                if style == G_reader_settings:readSetting("highlight_drawer") then
```

Comment:
Wouldn't something along the lines of highlight_drawing_style be a bit clearer?

Reply:
The same name as in doc settings.

Reply:
But I'd rather not *perpetuate* it.